lib_tflite_micro/src/tflite-xcore-kernels/xcore_strided_slice.cc

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022, XMOS Ltd, All rights reserved
```

Comment:
Could you please run clang-format on this file? Please let me know if you would like help with it.

Reply:
Done